### PR TITLE
Battle time with some styling fixes

### PR DESCRIPTION
**Pull request by erinnachen**
Make this a sweet 2 player battle for points. 
* Added a new floor layout for the 2 player version
* Those windups only jump in 2P mode
* Styled the screens a bit more
* Shouldn't have any issues switching between 1 and 2P modes
* I think I found the bug with the fruit being unable to be collected
* I deployed this branch on Heroku for your viewing enjoyment 

I'm missing sound fx 😕

### CODE REVIEW

**Comment by erinnachen** [comment]
@adriennedomingus This is what we worked on this afternoon minus the enemies jumping around. Maybe we just have them jump around for funsies. 😀
I think there's going to be issues with the game flow of what happens if you play a 1 player game and then want to play the 2p version and vice versa.

File: index.html [hunk 1]
```diff
@@ -8,81 +8,84 @@
...
+        <h3 class="screen-text">Bub (in the blue) wins!</h3>
         <h5 class="button new-single">New Single Player Game</h5>
         <h5 class="button new-double">New Two Player Game</h5>
       </div>
       <div id="end-game-lose-bob" class="hidden">
         <img src="images/bubble_bobble_logo.png" alt="bubble bobble logo" class="small-logo"/>
-        <h3>Game Over - Bub ran out of lives...</h3>
-        <h3>Bob wins!</h3>
+        <h3 class="screen-text">Game Over - Bub (in the blue) ran out of lives...</h3>
+        <h3 class="screen-text">Bob (in the green) wins!</h3>
```
Comment (erinnachen) [inline index.html]: I added this class for the font to be bigger

File: lib/fruits.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ Fruit.prototype.fall = function() {
 };
 
 Fruit.prototype.collectible = function() {
-  return (this.y > this.startingY + 10) && (this.status !== "collected");
+  return ((this.y > this.startingY + 10) || (this.y > this.canvas.height - this.height - 30)) && (this.status !== "collected");
 };
```
Comment (erinnachen) [inline lib/fruits.js]: I think this should handle the issue with randomly not being able to collect the fruit.

**Comment by adriennedomingus** [comment]
Yeaaaaah battle mode!

**Pull request merged**